### PR TITLE
[APPEND] functionabilities to update batch op (raw mode)

### PR DESCRIPTION
**Pull request by iariasleon**
#### regression test
```
                    SUMMARY:
 -------------------------------------------------------------------------------------------------------------------------------------------------
|                                          name                                               | passed | failed | skipped |  total  | duration(s) |
 -------------------------------------------------------------------------------------------------------------------------------------------------
| components\ngsiv2\batch_operations\batch_op_update\batch_op_update_action_type.feature      |     11 |      0 |      15 |      26 |       2.466 |
| components\ngsiv2\batch_operations\batch_op_update\batch_op_update_entities_id_type.feature |     62 |      0 |      28 |      90 |     238.959 |
| components\ngsiv2\batch_operations\batch_op_update\batch_op_update_headers.feature          |     54 |      0 |       1 |      55 |       8.312 |
 -------------------------------------------------------------------------------------------------------------------------------------------------
|  Date: Monday, November 14, 2016 18:51:38
 -------------------------------------------------
3 features passed, 0 failed, 0 skipped
127 scenarios passed, 0 failed, 44 skipped
662 steps passed, 0 failed, 240 skipped, 0 undefined
Took 4m9.737s

```

### CODE REVIEW

File: test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py [hunk 1]
```diff
@@ -60,5 +61,22 @@ def update_entities_in_a_single_batch_operation(context, op):
...
+    context.resp = context.cb.batch_update(queries_parameters, context.entities_accumulate, op)
     __logger__.info("...updated entities in a single batch operation")
+
+@step(u'update entities in a single batch operation "([^"]*)" in raw mode')
+def update_entities_in_a_single_batch_operation_in_raw_mode(context, op):
+    """
+    allows to create, update and/or delete several entities in a single batch operation in raw mode
+    used mainly with data type as boolean, dict, list, null, numeric, etc
+    :param context: It’s a clever place where you and behave can store information to share around. It runs at three levels, automatically managed by behave.
+    :param op: operations allowed (APPEND, APPEND_STRICT, UPDATE, DELETE)
```
Comment (fgalan) [inline test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py]: What about REPLACE actionType?

Comment (iariasleon) [inline test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py]: In the apiary reference in "Update Batch Operation" only show these actionTypes:
```
   actionType, to specify the kind of update action to do: either APPEND, APPEND_STRICT, UPDATE, DELETE.

```

Comment (iariasleon) [inline test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py]: the REPLACE actionType is tested and works correctly, So it should be included in the apiary. 
A new test is appended in this PR.

Comment (fgalan) [inline test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py]: Could you add a new issue (labels: bug, doc, P4) about aligning NGSIv2 doc for /v2/op/update operation to add REPLACE, pls?

Comment (fgalan) [inline test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py]: I have see the new test in 7bd2fb0 but I understand that this `:param op:` needs should also mention REPLACE.

Comment (iariasleon) [inline test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py]: Fixed in cc2f84b

**Comment by fgalan** [comment]
LGTM. Passing the ball to QA team for final LGTM and merge.

File: test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_headers.feature [hunk 2]
```diff
@@ -83,6 +83,52 @@ Feature: Batch operation - update using NGSI v2. "POST" - /v2/op/update plus pay
       | APPEND_STRICT |
 
   @happy_path
+  Scenario:  replace several entities with batch operations using NGSI v2
```
Comment (XavierVal) [inline test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_headers.feature]: Tip: Scenario title should start with a capital letter

File: test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_headers.feature [hunk 3]
```diff
@@ -150,14 +196,48 @@ Feature: Batch operation - update using NGSI v2. "POST" - /v2/op/update plus pay
...
       | attributes_value | 45          |
       | attributes_type  | kelvin      |
-    When update entities in a single batch operation "<operation>"
+    When update entities in a single batch operation "UPDATE"
+      | parameter | value     |
+      | options   | keyValues |
+    Then verify that receive a "No Content" http code
+
+  @happy_path
+  Scenario:  delete several entities  with batch operations) using NGSIv2
```
Comment (XavierVal) [inline test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_headers.feature]: Typo ")" or missing pair

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_headers.feature]: Fixed in ab2b171

File: test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_headers.feature (hunk 2 above)
Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_headers.feature]: This advice will be considered in a future refactor of all scenarios. thanks!!

**Comment by XavierVal** [comment]
LGTM

**Pull request merged**